Doc/library/__main__.rst

```diff
@@ -251,9 +251,9 @@ attribute will include the package's path if imported::
     >>> asyncio.__main__.__name__
     'asyncio.__main__'
 
-This won't work for ``__main__.py`` files in the root directory of a .zip file
-though.  Hence, for consistency, minimal ``__main__.py`` like the :mod:`venv`
-one mentioned above are preferred.
+This won’t work for ``__main__.py`` files in the root directory of a
```

Comment:
Was the change from a straight-up-and-down ASCII apostrophe (`won't`) to a "fancy" Unicode U+2019 RIGHT SINGLE QUOTATION MARK glyph (`won’t`) intentional here?

Reply:
Please change this back to an apostrophe.

Reply:
I think apostrophe is enough for consistency.

As I going through the python documentation, `won't` commonly appear as straight apostrophe.

Like:

- 1 `won't` appearance: https://docs.python.org/3/library/functions.html
- 2 `won't` appearances: https://docs.python.org/3/library/__main__.html